api-extractor/src/definitions/ApiItem.ts

```diff
@@ -59,7 +59,15 @@ export enum ApiItemKind {
   /**
    * A Typescript class constructor function.
    */
-  Constructor = 10
+  Constructor = 10,
+  /**
+   * A Typescript namespace.
+   */
+  Namespace = 11,
+  /**
+   * An abstract type field.
```

Comment:
>An abstract type field. [](start = 5, length = 23)

I wouldn't call this "abstract".  In C# a "field" is a language element that stores a simple data value (without any getter/setter function).

Seems like the TypeScript compiler API calls this a "BlockScopedVariable".

You could consider renaming ApiField -> ApiModuleVariable